foundation/foundation-upgrade/src/main/java/com/boozallen/aissemble/upgrade/migration/HelmChartsNameMigration.java

```diff
@@ -0,0 +1,80 @@
+package com.boozallen.aissemble.upgrade.migration;
```

Comment:
Recommend moving this migration and associated test files to the respective `v1_7_0` folders since this migration will not be needed in later upgrades.

Reply:
@carter-cundiff moved the migration and test files to `v1_7_0` directory